docs/estela/installation/helm-variables.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: page
+title: Variables Guide
+parent: Installation
+grand_parent: estela
+---
+
+# estela Helm Chart variables guide
+
+First, make a copy of `values.yaml.example` and rename it to `values.yaml`.
+Then, complete the following fields:
+
+_Note_: The values that should be used if the resources have been deployed locally are
+commented in the `values.yaml.example` file. If you do not need to define an optional 
+variable, fill its value with an empty string `""`.
+
+## Chart variables
+
+* _local_: Set this variable to `true` if estela is being deployed on local resources.
+
+* _hostIp_: (Optional) Fill this variable only if the variable _local_ has been set to 
+  `true`, this address is a reference to the host machine from minikube. Find it by running:
+  ```bash
+  $ minikube ssh 'grep host.minikube.internal /etc/hosts | cut -f1'
+  ```
+
+* _registryHost_: The registry host where are the images of the estela modules. If a local
+  registry is being used, do not forget to include the port.
+  
+* _awsRegistry_: Set this variable to `true` if the registry is being hosted in AWS.
+
+* _imagePullSecrets_: (Optional) Fill this variable only if the variable _awsRegistry_ 
+  has been set to `true`. Use the value `[ name: regcred ]`.
+
+* _nodeSelector_: (Optional) The name of the node on which estela will be installed in case
+  the Kubernetes cluster has multiple nodes. Use the format `{ roles: NODE_ROL_NAME }`.
+
+## estela module variables
+
+The variables that already have an assigned value should not be modified.
+
+* _<DB\_HOST>_: Host of the API module database.
+
+* _<DB\_PORT>_: Port of the API module database.
+
+* _<DB\_NAME>_: Database name used by the API module.
+
+* _<REGISTRY\_HOST>_: Address of the registry used by the API module.
```

Comment:
What would be the difference between this variable and the `registryHost` variable above?